moveit_core/collision_detection/include/moveit/collision_detection/collision_common.h

```diff
@@ -291,6 +295,7 @@ struct DistanceRequest
   bool compute_gradient;
 };
 
+/** \brief Representation of the distance information for a pair of objects. */
```

Comment:
Could you give a little more clarity here? Is this representation specific to FCL?

Reply:
No, it is not. I think it is supposed to be the general distance information format between two bodies independent of the checker.

Reply:
I see, I think my general confusion comes from the difference between DistanceResultData and DistanceResult, just saw that DistanceResult is just a wrapper for the former.